### ISSUE TITLE
why pred_c is detached

### ISSUE THREAD

**Issue by vtasStu** [lolrudy/GPV_Pose#7]
Hello, why is the [confidence of the rotation][1] all detach ？

```
 'Rot1_f': f_green_R.detach(),
 'Rot2_f': f_red_R.detach(),
```
[1]: https://github.com/lolrudy/GPV_Pose/blob/master/network/GPVPose.py#:~:text=%27Scale%27%3A%20p_s%2C-,%27Rot1_f%27%3A%20f_green_R.detach()%2C,%27Rot2_f%27%3A%20f_red_R.detach()%2C,-%7D

**Comment by lolrudy** [comment]
This prop_loss is designed to enforce the geometric constraint between the estimated pose and the observed point cloud. Since the confidence is already supervised in the fs_net_loss, we do not want to incorporate its gratitude in other losses.
